src/main/java/com/team766/controllers/OffsetPoint.java

```diff
@@ -0,0 +1,23 @@
+package com.team766.controllers;
+	/*
+	* This is a class to make an offset point for the PID controller to be able to reset encoders.
+	* You input the absoluteEncoderValue that equals the motorEncoderValue.
+	* @author Max Spier - 9/12/23
+	*/
+public class OffsetPoint {
```

Comment:
suggestion: create a toString override so that you can easily log this

Reply:
Why would we need to log the values here?

Reply:
in case anyone ever tries to println this class, it would use your toString instead of printing a cryptic address like "@3b24fa3"

Reply:
done